docs/reference/components/sinks/kafka.cue

```diff
@@ -19,8 +19,8 @@ components: sinks: kafka: {
 			batch: {
 				enabled:      true
 				common:       true
-				max_bytes:    null
 				max_events:   null
+				max_bytes:    null
```

Comment:
In all the others, you have taken out one; this one was just moved. Was this intentional?

Reply:
Aha, I didn't mean to move them, I just deleted them first and then re-added later; I must have re-added in a different order.